manifest: sdk-nrfxlib: nrf_fuel_gauge: Update to v2.0.0 (#28924)
nrfconnect/sdk-nrf: samples/pmic/native/npm13xx_fuel_gauge/README.rst

Overview
********

This sample allows to calculate the state of charge, time to empty, time to full, and state of health information from a battery on the development kit connected to the nPM1300 or nPM1304 PMIC.

> State of health is calculated based on battery charge cycles. This is a slow process. For testing purposes, one can do the following to accelerate the process:

Reviewer: ```suggestion
State of health is calculated based on battery charge cycles.
This is a slow process.
For testing purposes, you can accelerate the process as follows:
```

#. Use a battery model with a very low capacity that can be charged quickly, such as the 20 mAh model included in the nPM1304 version of this sample.
#. Configure the charging current to be as high as possible within the limits of your battery. Or as high as the PMIC allows when using a power supply or battery emulator instead of an actual battery.
#. Begin charging at state of charge less than 70%, and ensure that the charging cycle completes by reaching the configured termination voltage and termination current.
#. Ensure that charging completes before the battery state of charge reaches 100%, either by using a battery emulator, or configuring the charger termination voltage to be lower than the modeled battery termination voltage.

> #. Repeat charging cycles a minimum of two times to see any change in state of health. Note that state of health will only decrease when the state of charge is less than 100% when charging completes.

Reviewer: ```suggestion
* Use a battery model with a very low capacity that can be charged quickly, such as the 20 mAh model included in the nPM1304 version of this sample.
* Configure the charging current to be as high as possible within the limits of your battery or as high as the PMIC allows when using a power supply or battery emulator instead of an actual battery.
* Begin charging at state of charge less than 70%, and ensure that the charging cycle completes by reaching the configured termination voltage and termination current.
* Ensure that charging completes before the battery state of charge reaches 100%, either by using a battery emulator, or configure the charger termination voltage to be lower than the modeled battery termination voltage.
* Repeat the charging cycles at least two times to see any change in state of health.
  The state of health will only decrease when the state of charge is less than 100% when charging completes.
```

> State of health will also be calculated during partial charge cycles, but this is a slower process compared to full charge cycles.

Reviewer: ```suggestion
The state of health will also be calculated during partial charge cycles, but this is a slower process compared to full charge cycles.
```

> When the Zephyr Settings subsystem is enabled, the sample also demonstrates how to store the fuel gauge state in non-volatile memory and restore it on boot. Fuel gauge state can be stored using a shell command described in `Testing`_.

Reviewer: ```suggestion
When the Zephyr Settings subsystem is enabled, the sample also demonstrates how to store the fuel gauge state in non-volatile memory and restore it on boot.
You can store the fuel gauge state using a shell command described in `Testing`_.
```

.. code-block:: console

   PMIC device ok

>    V: 4.101, I: 0.000, T: 23.06, SoC: 93.09, TTE: nan, TTF: nan, SoH: 100.00

Reviewer 2: example printout missing `Chg`

.. code-block:: console

   PMIC device ok

>    V: 4.101, I: 0.000, T: 23.06, SoC: 93.09, TTE: nan, TTF: nan, SoH: 100.00, Chg: 0.00

Reviewer 3: Cycle count missing ?
Chg or charged energy can also be optional

Reviewer 4: I can add cycle count

   SoC     State of Charge  Percent
   TTE     Time to Empty    Seconds (may be NaN)
   TTF     Time to Full     Seconds (may be NaN)
   SoH     State of health  Percent
   Chg     Charged energy   milliAmps (resets after a full charge cycle)
   ======  ===============  ==================================================

To store the fuel gauge state in non-volatile memory, use the following shell command:

.. code-block:: console

   $ fuel_gauge_state_store
   Storing state after next update

> Fuel gauge state will be stored in non-volatile memory after the next measurement has been made, and restored on the next boot.

Reviewer: ```suggestion
The fuel gauge state will be stored in non-volatile memory after the next measurement has been performed, and restored on the next boot.
```

   SoC     State of Charge  Percent
   TTE     Time to Empty    Seconds (may be NaN)
   TTF     Time to Full     Seconds (may be NaN)

>    SoH     State of health  Percent

Reviewer 3: ( maximum available battery capacity)

>    Chg     Charged energy   milliAmps (resets after a full charge cycle)

Reviewer 3: milliAmpere hours ?

Reviewer 4: Ah, yes. Fixing this
